packages/apollo-server-integration-testsuite/src/ApolloServer.ts

```diff
@@ -2251,11 +2254,6 @@ export function testApolloServer<AS extends ApolloServerBase>(
     describe('Response caching', () => {
       let clock: FakeTimers.InstalledClock;
       beforeAll(() => {
-        // These tests use the default InMemoryLRUCache, which is backed by the
-        // lru-cache npm module, whose maxAge feature is based on `Date.now()`
-        // (no setTimeout or anything like that). So we want to use fake timers
-        // just for Date. (Faking all the timer methods messes up things like a
-        // setImmediate in ApolloServerPluginDrainHttpServer.)
         clock = FakeTimers.install({ toFake: ['Date'] });
```

Comment:
Can we just stop using this then?

Reply:
No because the plugin actually uses `Date` to calculate the `age` header, so this is actually still useful (and required for testing that)

Reply:
Ah. Then update the comment instead of deleting it...

I do wish we understood why lru-cache doesn't seem to work with fake-timers. I do worry there's an actual bug there.

Reply:
We looked into this for a while. It seems like the combination of lru-cache + fake-timers + Jest is messed up... something maybe about Jest's fake globals combined with lru-cache hanging on to a copy of `performance` early on, perhaps...